smartsim/_core/entrypoints/colocated.py

```diff
@@ -201,8 +202,10 @@ def main(
     # we generally want to catch all exceptions here as
     # if this process dies, the application will most likely fail
     try:
-        process = psutil.Popen(cmd, stdout=PIPE, stderr=STDOUT)
-        DBPID = process.pid
+        with open("colo_orch_output.txt", "w", encoding="utf-8") as file:
```

Comment:
I kind of think we dump this to `/dev/null` unless we're in debug mode. Otherwise, to help with debugging, it would be useful to include the hostname so that we can separate out the logs from each node. Also, please rename the extension to .log.

Reply:
I was not familiar with dumping to `/dev/null` but I agree with the suggestion!